core/templates/dev/head/pages/review-test-page/review-test-page.controller.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controllers for the review tests.
+ */
+oppia.constant('INTERACTION_SPECS', GLOBALS.INTERACTION_SPECS);
+oppia.constant('REVIEW_TEST_DATA_URL', '/review_test_handler/data/<story_id>');
+
+require(
+  'components/common-layout-directives/common-elements/' +
+  'background-banner.directive.ts');
+require(
+  'components/question-directives/question-player/' +
+  'question-player.directive.ts');
+require('objects/objectComponentsRequiresForPlayers.ts');
+require('pages/review-test-page/review-test-engine.service.ts');
+require('services/AlertsService.ts');
+require('services/contextual/UrlService.ts');
+
+oppia.controller('ReviewTest', [
```

Comment:
@YashJipkate I have a query for you: should we update the wiki to add component directives instead of new standalone controllers. Also, should @sophiewu6 be adding a component directive or are you planning to make changes to this later in your work?

Reply:
Yes, thanks @ankita240796 for pointing this out. 

@sophiewu6 Could you please use component directives instead as mentioned [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#component-directives)? You can refer to the already converted pages to get an idea. Such old style controllers are discouraged and we are removing them from our codebase.